### DISCUSSION TITLE
How do I find the following forecast

### DISCUSSION THREAD

**Discussion by ernieratcliff** [weather-gov/api#355] [Q&A]
Hello,

I'm trying to use api.weather.com to find the forecast that is equivalent to the following link https://marine.weather.gov/MapClick.php?x=320&y=176&site=mhx&zmx=&zmy=&map_x=320&map_y=176#.XqI3JGhKiiN

I've looked at the zone and search the various grid points like this but you can tell this result does not return the same forecast as in the link above.  No Wave info etc.  
https://api.weather.gov/gridpoints/MHX/47,17/forecast

What is the right way to lookup this forecast?

Thanks,
Ernie

**Comment by StephenClouse** [comment]
The API currently does not support marine forecasts. Sorry about that.

**Comment by ernieratcliff** [comment]
Thanks Stephen for the quick response.  Would the older services support
marine forecasts?

**Comment by StephenClouse** [comment]
Yes, the existing legacy services will continue to support marine.

We do intend to add full marine support to API at a later date, but it's just one of those things we had to push back due to priorities.

**Comment by ernieratcliff** [comment]
Understood...thanks again for the info

**Comment by ernieratcliff** [comment]
sorry one thought.  It might be good to state exactly what the API supports
on the documentation website.  It could be I just missed this but from just
playing around with it I never got that it didn't support marine
